test/traversals/bfs.jl

```diff
@@ -89,4 +89,26 @@ import LightGraphs: TreeBFSVisitorVector, bfs_tree!, tree
       h = blkdiag(g, g)
       @test @inferred(bipartite_map(h)) == Vector{T}([ones(T, 10); 2 * ones(T, 10); ones(T, 10); 2 * ones(T, 10)])
     end
+
+    let g = Graph(6)
+        d = nv(g)
+        for (i,j) in [(1,2), (2,3), (2,4),(4,5), (3,5)]
+            add_edge!(g,i,j)
```

Comment:
minor: if you're editing this file anyway, space the tuples consistently, and put spaces between args in `add_edge!`